js/core/examples.js

```diff
@@ -14,7 +14,7 @@ define(
     function (pubsubhub, css) {
         var makeTitle = function (conf, $el, num, report) {
             var txt = (num > 0) ? " " + num : ""
-            ,   $tit = $("<div class='example-title'><span>Example" + txt + "</span></div>");
+            ,   $tit = $("<div class='example-title'><span>" + conf.l10n.example + txt + "</span></div>");
```

Comment:
Sorry, to clarify, don't change things in "js/core" or "js/w3c". If you change those, then we will no longer be able to get updates from the W3C upstream.